build/deps/build_deps.jsonc

```diff
@@ -93,6 +93,15 @@
       "repo": "rules_pkg",
       "file_regex": "\\.tar\\.gz$"
     },
+    {
+      // Needed for running the latest Node.js due to aspect_rules_js
+      "name": "rules_nodejs",
+      "type": "github_release",
+      "owner": "bazel-contrib",
+      "repo": "rules_nodejs",
+      "file_regex": "\\.tar\\.gz$",
+      "freeze_version": "v6.3.2"
```

Comment:
- [ ] No need to freeze this – later versions will work just as well

Reply:
Thanks! Removed it.